src/test/resources/network/brightspots/rcv/test_data/dominion_no_precinct_data/dominion_no_precinct_data_summary.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
I don't think this file is necessary, right?

Reply:
ack!